Makefile

```diff
@@ -168,6 +220,8 @@ help:
 	$(call help_once,NOVENDOR            --  Avoid to self detect and consume the vendored headers)
 	$(call help_once,                        shipped with libbpf-sys.)
 	$(call help_once,RA                  --  Applies to check and clippy and runs those targets with the options needed)
-	$(call help_once,                        for rust-analyzer. When $$(RA) is used, $$(V) becomes ineffective.)
+	$(call help_once,                        for rust-analyzer. When $$(RA) is used $$(V) becomes ineffective.)
```

Comment:
`rust-analyzer` is still here, but not sure if this is intended. (it's probably OK but just want to make sure).

Reply:
yes, this is intended (not a target)